src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -26,14 +26,14 @@
 }
 
 // UI Enhancer "Enlarge text" input/button options
-.fl-text-larger input,
-.fl-text-larger button,
-.fl-text-larger .fl-input-enhance {
+.fl-input-enhanced input,
```

Comment:
Perhaps we should do this for [form elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Forms) in general.